database/sqlite/sqlite_db_migration.c

```diff
@@ -315,6 +321,25 @@ static int do_migration_v10_v11(sqlite3 *database, const char *name)
     return 0;
 }
 
+static int do_migration_v11_v12(sqlite3 *database, const char *name)
+{
+    char sql[2048];
+    int rc = 0;
+
+    netdata_log_info("Running \"%s\" database migration", name);
+
+    if (table_exists_in_database("health_log_detail") && !column_exists_in_table("health_log_detail", "summary") &&
+        table_exists_in_database("alert_hash") && !column_exists_in_table("alert_hash", "summary"))
+        rc = init_database_batch(database, &database_migrate_v11_v12[0]);
+
+    if (!rc) {
+        snprintfz(sql, 2047, "UPDATE health_log_detail SET summary = (select name from health_log where health_log_id = health_log_detail.health_log_id);");
```

Comment:
Is really necessary to copy a string to a vector? Can we use only a `define` here?